xclim/core/calendar.py

```diff
@@ -527,6 +527,7 @@ def percentile_doy(
     p.attrs["window"] = window
     p.attrs["alpha"] = alpha
     p.attrs["beta"] = beta
+    p.attrs["per_base_thresh"] = per
```

Comment:
Isn't this information already available in the `percentiles` coordinate? The issue here is that I'm not sure the netCDF  backend (or others) will elegantly handle a list passed as attribute. Even more problematic is if the user passed a `ndarray` or a `DataArray`.

Reply:
True, I forgot it's a already there.